include/dlaf/lapack/gpu/add.h

```diff
@@ -0,0 +1,38 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#ifdef DLAF_WITH_GPU
+
+#include "dlaf/gpu/blas/api.h"
+#include "dlaf/types.h"
+
+#include <blas.hh>
+#include <whip.hpp>
+
+namespace dlaf::gpulapack {
+
+template <class T>
+void add(const blas::Uplo uplo, const SizeType m, const SizeType n, const T& alpha, const T* a,
+         const SizeType lda, T* b, const SizeType ldb, const whip::stream_t stream);
+
+#define DLAF_CUBLAS_ADD_ETI(kword, Type)                                                                \
```

Comment:
not sure I get why it is called CUBLAS...but it is not the topic of this PR anyway

Reply:
I simply copied https://github.com/eth-cscs/DLA-Future/blob/master/include/dlaf/lapack/gpu/lacpy.h#L27 😇